.github/workflows/ci_test-full.yml

```
@@ -33,8 +33,7 @@ jobs:
 
     - name: Update Pip
       run: |
-        # todo: unfreeze PIP after resolving minimal dependencies
-        pip install --quiet "pip==20.1" --upgrade --user  # needed for get pip cacher folder
+        pip install --quiet "pip>=20.1" --upgrade --user  # needed for get pip cacher folder
```

```suggestion
        pip install --quiet "pip>=20.3" --upgrade --user  # needed for get pip cacher folder
```
maybe try some higher version than 20.1?